packages/flutter_driver/lib/android_testing.dart

```diff
@@ -0,0 +1,14 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// WARNING: These APIs are only supported internally for testing the flutter
+/// framework.
```

Comment:
We either support it or we don't have it, we don't have a concept of half-public APIs.

Reply:
(This is because regardless of what we say, people will depend on whatever we publish.)